BridgeAppSDK/SBAProfileInfoForm.swift

```diff
@@ -145,38 +148,48 @@ public struct SBAProfileInfoOptions {
                                            answerFormat: answerFormat,
                                            optional: false)
                 formItem.placeholder = Localization.localizedString("EMAIL_FORM_ITEM_PLACEHOLDER")
-                formItems += [formItem]
-                
-                let passwordAnswerFormat = ORKAnswerFormat.textAnswerFormat()
-                passwordAnswerFormat.multipleLines = false
-                passwordAnswerFormat.secureTextEntry = true
-                passwordAnswerFormat.autocapitalizationType = .None
-                passwordAnswerFormat.autocorrectionType = .No
-                passwordAnswerFormat.spellCheckingType = .No
-                
-                let passwordFormItem = ORKFormItem(identifier: option.rawValue,
+                formItems.append(formItem)
+            
+            case .password:
+                let answerFormat = ORKAnswerFormat.textAnswerFormat()
+                answerFormat.multipleLines = false
+                answerFormat.secureTextEntry = true
+                answerFormat.autocapitalizationType = .None
+                answerFormat.autocorrectionType = .No
+                answerFormat.spellCheckingType = .No
+
+                let formItem = ORKFormItem(identifier: option.rawValue,
                                                    text: Localization.localizedString("PASSWORD_FORM_ITEM_TITLE"),
-                                                   answerFormat: passwordAnswerFormat,
+                                                   answerFormat: answerFormat,
                                                    optional: false)
-                passwordFormItem.placeholder = Localization.localizedString("PASSWORD_FORM_ITEM_PLACEHOLDER")
-                formItems += [passwordFormItem]
+                formItem.placeholder = Localization.localizedString("PASSWORD_FORM_ITEM_PLACEHOLDER")
+                formItems.append(formItem)
                 
                 // confirmation
                 if (surveyItemType == .account(.registration)) {
-                    let confirmIdentifier = "\(formItem.identifier).\(SBARegistrationStep.confirmationIdentifier)"
-                    let confirmFormItem = passwordFormItem.confirmationAnswerFormItemWithIdentifier(confirmIdentifier,
-                                                                                                    text: Localization.localizedString("CONFIRM_PASSWORD_FORM_ITEM_TITLE"),
-                                                                                                    errorMessage: Localization.localizedString("CONFIRM_PASSWORD_ERROR_MESSAGE"))
+                    
+                    // If this is a registration, go ahead and set the default password verification
+                    let minLength = SBARegistrationStep.defaultPasswordMinLength
+                    let maxLength = SBARegistrationStep.defaultPasswordMaxLength
```

Comment:
default min and max passwords are used in the protocol which can't have static variables defined on it. Therefore, I put them on the registration step.